free-courses-fi.md

```diff
@@ -20,3 +21,8 @@
 ### Python
 
 * [Helsingin yliopiston Python-ohjelmointikurssi](https://linkki.github.io/python2017) - Ilmainen verkkokurssi
+
+
+### Web Development
+
+* [Full stack open](https://fullstackopen.com/) - University of Helsinki
```

Comment:
Hello,

```suggestion
* [Full stack open](https://fullstackopen.com) - University of Helsinki
```

Removing the trailing slash as per [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) guidelines should make the Travis CI build pass.

> on root domains, strip the trailing slash: http://example.com instead of http://example.com/

Reply:
Done, thank you! 🙂

Reply:
@baciucristian yeah anytime!